spec/shinq_spec.rb

```diff
@@ -9,6 +9,10 @@ def shinq_class
 
 describe Shinq do
   subject { shinq_class }
+  after do
+    Shinq.clear_all_connections!
+    shinq_class.clear_all_connections!
```

Comment:
These two lines seem the same codes but actually are not.
Former one clears connections stored in `Shinq`, while latter one clears connections stored in duped `Shinq`.

The reason why the former one was required in addition to the latter one is that `shinq_class` was called many times and those duped classes shares the same object `@connections = {}` which were defined in `Shinq` in prior test. Therefore, `Shinq.clear_all_connections!` cleared not only `Shinq` connections but also all the connections defined in `Shinq.dup`.

I fixed duping strategy on rspec to resolve this problem. Now the duped class is consistent within each example with all the instance variables initialized (removed).